bin/cheri_bench/Makefile

```diff
@@ -2,11 +2,15 @@
 
 .include <src.opts.mk>
 
+.if !${MACHINE_ABI:Mpurecap}
+NEED_COMPAT=	CHERI
+.include <bsd.compat.mk>
```

Comment:
Is this program ever used? If not I think we should just delete it.

Reply:
Gone with the libcheri removal